pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -396,7 +399,11 @@ func (p *backendImpl) processIDPResponseForImplicitFlowWithIDToken(ctx context.C
 		return nil, errors.New("no id_token field found in response")
 	}
 
-	authResp, err := p.verifyIDToken(ctx, rawIDToken, verifyNonce)
+	// For the implicit flow, according to the spec an access token _should_ be sent which should be valid for
+	// calling the user info endpoint, however we currently cannot take that assumption. Since the indirection
+	// for the ID token claims not containing groups and requiring fetching the user info endpoint claims is only
+	// observed for the code flow, we for the time being skipped the indirection in the implicit flow.
```

Comment:
It's not quite clear what you mean under "indirection". Is it that extra round trip to /userinfo?

Anyway, I'd suggest to actually pass `responseData.Get("access_token")` into the function leaving a note that we might not have access_token in the IdP response but it's fine.